lua/entities/ent_jack_boundingmine/cl_init.lua

```diff
@@ -39,26 +39,28 @@ function ENT:Draw()
     local Ang4 = self:GetAngles()
     local Up = self:GetUp()
 
+    local selfTbl = self:GetTable()

```

Comment:
It might be worth plucking out some of the meta functions as locals - I think that's still more performant.